specs/referrer-policy/index.src.html

```diff
@@ -535,24 +537,24 @@ <h3 id="referrer-policy-delivery-referrer-attribute">Delivery
     <dd><a><code>No Referrer</code></a></dd>
     <dt>origin</dt>
     <dd><a><code>Origin</code></a></dd>
+    <dt>no-referrer-when-downgrade</dt>
+    <dd><a><code>No Referrer When Downgrade</code></a></dd>
+    <dt>origin-when-cross-origin</dt>
+    <dd><a><code>Origin When Cross-Origin</code></a></dd>
     <dt>unsafe-url</dt>
     <dd><a><code>Unsafe URL</code></a></dd>
   </dl>
 
-  A policy delivered via a <code>referrer</code> attribute on an element
+  Note: A policy delivered via a <code>referrerpolicy</code> attribute on an element
   takes precedence over the policy defined for the whole document via CSP or
   a <a element>meta</a> element.
 
-  ISSUE: If an <code><a element>a</a></code>
+  Note: If an <code><a element>a</a></code>
   or <code><a element>area</a></code> element includes both
-  a <code>referrer</code> attribute as well as
-  a <a href="https://html.spec.whatwg.org/#link-type-noreferrer"><code>noreferrer</code>
-  link type</a>, should the policy of the <code>referrer</code> attribute
-  take precedence over the link type, as suggested
-  by <a href="https://lists.w3.org/Archives/Public/public-webappsec/2015Feb/0235.html">Martin
-  Thomson</a>, or should we take the more conservative approach as suggested
-  by <a href="https://lists.w3.org/Archives/Public/public-webappsec/2015Feb/0283.html">Brian
-  Smith</a> and honor the <code>noreferrer</code> link type?
+  a <code>referrerpolicy</code> attribute as well as
+  a <a><code>noreferrer</code></a> link type then the <a><code>noreferrer</code></a>
+  link type will take precedence and the <a><code>No Referrer</code></a> policy
+  takes effect.
```

Comment:
can you add some text to what the referrer policy applies? Something like the request?

I guess for <a> and <area> it's the navigation triggered by the user agent when clicking on it.

for <img> it's the request for src

what about <iframe>, is it for loading the iframe only, or will it also set the policy for the iframe's global context?